lib/anvil.ex

```diff
@@ -1,2 +1,63 @@
 defmodule Anvil do
+  defmodule UndefinedFixture do
+    defexception [:message]
+
+    def exception(fixture_name) do
+      message = "No fixture defined for #{inspect fixture_name}"
+      %UndefinedFixture{message: message}
+    end
+  end
+
+  defmacro __using__(_opts) do
+    quote do
+      @before_compile unquote(__MODULE__)
+
+      def assoc(opts, fixture_name) do
```

Comment:
Thoughts on putting these methods outsude if `__using__` or a separate module so you can document them? https://github.com/BlakeWilliams/Elixir-Slack/blob/master/lib/slack.ex#L75